packages/shared/features/features.js

```diff
@@ -1,34 +1,7 @@
-const featureFlags = {
-    onboarding: {
-        enabled: true,
-        iota: {
-            enabled: false,
-            mainnet: {
-                enabled: false,
-            },
-            devnet: {
-                enabled: false,
-            },
-            'private-net': {
-                enabled: false,
-            },
-        },
-        shimmer: {
-            enabled: true,
-            claimRewards: {
-                enabled: false,
-            },
-            mainnet: {
-                enabled: false,
-            },
-            devnet: {
-                enabled: true,
-            },
-            'private-net': {
-                enabled: false,
-            },
-        },
-    },
+import onboardingFeatures from './onboardingFeatures'
+
+const features = {
+    onboarding: onboardingFeatures,
```

Comment:
if we extract the `onboardingFeatures` into an own file, i think we should do that also for the other

Reply:
Yes will do, but limited the scope of this pr to onbaording